model/telemetry/registry.yaml

```diff
@@ -81,3 +81,57 @@ groups:
         brief: >
           The version string of the auto instrumentation agent or distribution, if used.
         examples: ["1.2.3"]
+      - id: telemetry.sdk.processor.type
+        type:
+          members:
+            - id: batch_span
```

Comment:
Any reason to have `span` in the id?
Since `telemetry.sdk.processor.type` with `batch` and `simple` (plus any custom value) would work for any processor while the metric name would contain the signal name.

Reply:
I was assuming that attributes should have a clear, unambiguous definition even when considering them outside of their current use cases: Using just `batching` or `simple` for `telemetry.sdk.processor.type` in isolation would be ambiguous: It could be a log or a span processor.

So for example if we later decide to add a `telemetry.sdk.processor.cpu_time` metric to quantify the overhead, `batch` or `simple` would be ambiguous for `telemetry.sdk.processor.type`.

So if I'm wrong here and we can basically say that this attribute needs to be used in contexts where the type of signal (span, log, metric) is known, I'd propose to go even further and combine `telemetry.sdk.processor.type` with `telemetry.sdk.exporter.type` to a single `telemetry.sdk.component.type` definition.